tests/phpunit/Unit/iCalendar/DateParserTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace SRF\Tests\iCalendar;
+
+use SRF\iCalendar\DateParser;
+
+/**
+ * @covers \SRF\iCalendar\DateParser
+ * @group semantic-result-formats
+ *
+ * @license GNU GPL v2+
+ * @since 3.1
+ *
+ * @author mwjames
+ */
+class DateParserTest extends \PHPUnit_Framework_TestCase {
+
+	public function testCanConstruct() {
+
+		$this->assertInstanceOf(
+			DateParser::class,
+			new DateParser()
+		);
+	}
```

Comment:
I think this test is better omitted

Reply:
I have testing the constructor for a while now and it served me well,it  even found some issues in other tests.